app/Commands/StartCommand.php

```diff
@@ -12,7 +12,7 @@ class StartCommand extends Command
 {
     use InitializesCommands;
 
-    protected $signature = 'start {containerId?}';
+    protected $signature = 'start {containerId?*} {--all}';
```

Comment:
What's the `*`? If it captures *anything* passed (so someone could pass more than one), wouldn't we need to update the consuming code to handle that? Thanks!

Reply:
Initially added the asterisk because `takeout start --all` would still consider `--all` to be a `containerId` and fail. Not sure what I did wrong there. 

You are right though, I will update my PR to handle multiple containers.